alpha/lib/model/EntryServerNodePeer.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+
+/**
+ * Skeleton subclass for performing query and update operations on the 'entry_server_node' table.
+ *
+ * 
+ *
+ * You should add additional methods to this class to meet the
+ * application requirements.  This class will only be generated as
+ * long as it does not already exist in the output directory.
+ *
+ * @package Core
+ * @subpackage model
+ */
+class EntryServerNodePeer extends BaseEntryServerNodePeer {
+
+	// cache classes by their type
+	protected static $class_types_cache = array();
+
+	/* (non-PHPdoc)
+	 * @see BaseUserEntryPeer::setDefaultCriteriaFilter()
+	 */
+	public static function setDefaultCriteriaFilter()
```

Comment:
setDefaultCriteriaFilter doesn't add any criterion to the criteria, you can remove the entire method.